src/views/assets/Asset/AssetCreateUpdate/components/AssetAccounts.vue

```diff
@@ -251,4 +251,13 @@ export default {
     height: 100%;
   }
 }
+
+.detail-drawer {
+  ::v-deep {
+    .el-drawer__header {
+      border-bottom: none;
+      padding-bottom: 1px;
+    }
+  }
+}
 </style>
```

Comment:
I would need some specific files to analyze. Could you please provide the relevant CSS file or the source for this template?